cppcache/integration-test/resources/system.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # All the configurable parameters.
 statistic-sample-rate=700s
-statistic-sampling-enabled=false
+statistic-sampling-enabled=true
```

Comment:
I don't understand this change.  Doesn't this change the value to true for _all_ integration tests now?  I thought only the expiration tests required this, i.e. the two places where you're now explicitly enabling them in test code.

Reply:
No it’s used by a single integration test.

Reply:
Ugh.  I'd ask that we rename this file to reflect that it's test-case-specific, but I think time is better spent just getting rid of the old tests in general.  Thanks for the clarification.

Reply:
Yeah, when I need more time in C++ again after ACE is gone...